### PR TITLE
Move selectors and actions files to the store folder

### PR DESCRIPTION
**Pull request by youknowriad**
This does nothing aside moving the selectors and actions to the `store` folder. This gives the `editor` module a nice folder structure.

I'd appreciate a quick review/merge to avoid rebases.

### CODE REVIEW

File: editor/store-persist.js [hunk 1]
```diff
@@ -1,63 +0,0 @@
-/**
```
Comment (aduth) [inline editor/store-persist.js]: I guess this was a dead file?

**Review by aduth — APPROVED** [review]
Might run into some issues with stale branches merging based on the old paths, but I assume we can fix these as the issues occur.

File: editor/store-persist.js (hunk 1 above)
Comment (youknowriad) [reply editor/store-persist.js]: Yep 👍

**Comment by gziolo** [comment]
Nice 👍

**Pull request merged**